### PR TITLE
3.3

### PR DESCRIPTION
**Pull request by chives**

### CODE REVIEW

File: features/fixtures/project/config/packages/framework.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+parameters:
+    locale: en
```
Comment (szymach) [inline features/fixtures/project/config/packages/framework.yaml:2]: Is this used anywhere?

**Review by szymach — APPROVED** [review]

File: features/fixtures/project/config/packages/framework.yaml (hunk 1 above)
Comment (chives) [reply features/fixtures/project/config/packages/framework.yaml:2]: yes

**Comment by chives** [comment]
@fsi-open

**Comment by chives** [comment]
@fsi-open

**Review by szymach — APPROVED** [review]

**Review by rn0 — APPROVED** [review]

**Pull request merged**